site/app/views/NavigationView.php

```diff
@@ -694,4 +694,13 @@ public function deleteGradeableForm() {
         return $this->core->getOutput()->renderTwigTemplate("navigation/DeleteGradeableForm.twig");
     }
 
+    public function listNotifications($notifications, $show_all) {
```

Comment:
We could probably put this in a `NotificationsView` class or just call this code in-line where this function gets invoked (`NavigationController`).  It doesn't make much sense to have it be in the Navigation View